File: apps/employee-service-server/src/APIs/Employee/Dtos/EmployeeWhereInput.cs
using EmployeeService.Core.Enums;

namespace EmployeeService.APIs.Dtos;

public class EmployeeWhereInput
{
    public string? City { get; set; }

    public string? Country { get; set; }

    public DateTime? CreatedAt { get; set; }

    public string? Designation { get; set; }

    public GenderEnum? Gender { get; set; }

    public string? Id { get; set; }

    public string? Name { get; set; }

    public DateTime? UpdatedAt { get; set; }
}

File: apps/employee-service-node-admin/src/api/employee/EmployeeWhereInput.ts
import { StringNullableFilter } from "../../util/StringNullableFilter";
import { StringFilter } from "../../util/StringFilter";

export type EmployeeWhereInput = {
  city?: StringNullableFilter;
  country?: StringNullableFilter;
  designation?: StringNullableFilter;
  gender?: "Option1";
  id?: StringFilter;
  name?: StringNullableFilter;
};
